src/Controls/tests/DeviceTests/Elements/SearchBar/SearchBarTests.cs

```diff
@@ -10,6 +12,21 @@ namespace Microsoft.Maui.DeviceTests
 	[Category(TestCategory.SearchBar)]
 	public partial class SearchBarTests : ControlsHandlerTestBase
 	{
+		void SetupBuilder()
+		{
+			EnsureHandlerCreated(builder =>
+			{
+				builder.ConfigureMauiHandlers(handlers =>
+				{
+					handlers.AddHandler(typeof(Toolbar), typeof(ToolbarHandler));
+					handlers.AddHandler(typeof(SearchBar), typeof(SearchBarHandler));
+					handlers.AddHandler(typeof(NavigationPage), typeof(NavigationViewHandler));
+					handlers.AddHandler<Page, PageHandler>();
+					handlers.AddHandler<Window, WindowHandlerStub>();
```

Comment:
```suggestion
					handlers.AddHandler(typeof(SearchBar), typeof(SearchBarHandler));
```
We only need to add the SearchBar handler, since that's the only one needed for the test!

Reply:
This code is removed now